### PR TITLE
webpod 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by antonmedv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/webpod.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/webpod --version")
```
[the remark this answers is not in the document]
Comment (p-linnane) [reply Formula/webpod.rb:18]: There's nothing we can run to verify functionality even if it would fail?

[the remark this answers is not in the document]
Comment (antonmedv) [reply Formula/webpod.rb:18]: Well,it connects to a remote host and runs commands there. Nothing is done locally.

[the remark this answers is not in the document]
Comment (p-linnane) [reply Formula/webpod.rb:18]: Right. Can we pass it a fake hostname and verify that it fails to connect?

**Review by chenrui333 — APPROVED** [review]

File: Formula/webpod.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/webpod.rb:18]: 👍

File: Formula/webpod.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require "language/node"
+
+class Webpod < Formula
+  desc "Deploy websites and apps anywhere"
+  homepage "https://github.com/webpod/webpod"
+  url "https://registry.npmjs.org/webpod/-/webpod-1.0.0.tgz"
```
Comment (chenrui333) [inline Formula/webpod.rb:6]: one nitpick comment, your release tags (https://github.com/webpod/webpod/releases) do not match with npmjs artifact

File: Formula/webpod.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+require "language/node"
+
+class Webpod < Formula
+  desc "Deploy websites and apps anywhere"
+  homepage "https://github.com/webpod/webpod"
```
Comment (chenrui333) [inline Formula/webpod.rb:5]: actually you can also update this url to `https://webpod.dev/`

File: Formula/webpod.rb (hunk 2 above)
Comment (antonmedv) [reply Formula/webpod.rb:6]: True! Thanks for noticing. Will fix it.

File: Formula/webpod.rb (hunk 3 above)
Comment (antonmedv) [reply Formula/webpod.rb:5]: Done.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@antonmedv, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**